pypeit/scripts/setup_coadd2d.py

```diff
@@ -97,15 +99,20 @@ def main(args):
             # based on the parameter value, then try to base it on the parent
             # directory of the provided pypeit file.  The latter is critical to the
             # vet_test in the dev-suite.
-            sci_dir = Path(par['rdx']['redux_path']).resolve() / par['rdx']['scidir']
-            if not sci_dir.exists():
-                sci_dir = Path(args.pypeit_file).resolve().parent / par['rdx']['scidir']
+            sci_dirs = [Path(par['rdx']['redux_path']).resolve() / par['rdx']['scidir']]
+            if not sci_dirs[0].exists():
+                sci_dirs = [Path(args.pypeit_file).resolve().parent / par['rdx']['scidir']]
 
-        if not sci_dir.exists():
-            msgs.error(f'Science directory not found: {sci_dir}\n')
+        sci_dirs_exist = [sc.exists() for sc in sci_dirs]
+        if not np.all(sci_dirs_exist):
+            msgs.error(f'One or more Science directories not found.\n')
```

Comment:
Nit-picking...  Particularly given that you define `sci_dirs_exist`, I would print the science directories that do not exist in the error.